modules/about-the-single-model-serving-platform.adoc

```diff
@@ -24,36 +24,14 @@ Automated installation:: If you have not already created a `ServiceMeshControlPl
 
 Manual installation:: If you have already created a `ServiceMeshControlPlane` or `KNativeServing` resource on your OpenShift cluster, you _cannot_ configure the {productname-long} Operator to install KServe and configure its dependencies. In this situation, you must install KServe manually.
 
-When you have installed KServe, you can use the {productname-short} dashboard to deploy models using pre-installed or custom model-serving runtimes. 
+When you have installed KServe, you can use the {productname-short} dashboard to deploy models using preinstalled or custom model-serving runtimes. 
```

Comment:
I think installed is not the right term here ..imo we should use `enabled`

Reply:
Because I only changed this sentence to correct a style mistake, I think this request is slightly out of scope for this review. I also believe changing "install" in relation to KServe would be a bigger request, because we use "install" throughout our documentation: https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2-latest/html-single/serving_models/index#manually-installing-kserve_serving-large-models
Happy to open a separate ticket to track. WDYT? @tarukumar